client/webpack.config.node.js

```diff
@@ -1,5 +1,5 @@
 /**
- * **** WARNING: No ES6 modules here. Not transpiled! ****
+ * WARNING: No ES6 modules here. Not transpiled! ****
```

Comment:
Nit of the day: if we're removing the leading `****` should we also remove the trailing one?
```suggestion
 * WARNING: No ES6 modules here. Not transpiled!
```